src/Relation/Binary/Properties/TotalOrder.agda

```diff
@@ -13,7 +13,9 @@ open import Relation.Binary.Structures using (IsTotalOrder)
 module Relation.Binary.Properties.TotalOrder
   {t₁ t₂ t₃} (T : TotalOrder t₁ t₂ t₃) where
 
-open TotalOrder T
+-- issue #1214: locally to this module, we rename the relation _≰_, so
+-- that we can deprecate it here, yet re-export it elsewhere
+open TotalOrder T renaming (_≰_ to _≰A_)
```

Comment:
Same problem here with the `A`. Why are we renaming it at all?

Reply:
This module used to export this symbol `_≰_`. And this behaviour, likewise for `StrictPartialOrder` with `\<n`, was an anomalous special case, which this PR is now systematising. So... to maintain backwards compatibility but with deprecation to force users to rewrite their imports, this seemed the only way to achieve that end. Otherwise the symbol we eventually want to have ends up being deprecated. Or, as above: is there a better way to achieve this?

So library clients who try to import that symbol from here should be given a deprecation warning if they do, and directed to `import` it form the correct module (or even, not at all, as it should simply come for free from `Bundles`)